spec/migrations/update_progress_calculation_spec.rb

```diff
@@ -858,14 +863,14 @@ def expect_migrates(from:, to:)
           .to eql [nil, 50]
       end
 
-      it "creates a journal for the work package transitioning ∑ % complete from 90 to 50" do
+      it "creates a journal for the work package transitioning ∑ % complete from 90% to 20%" do
```

Comment:
Should this spec also contain an expectation to a journal entry of setting the remaining work to the correct value?

Reply:
This context was testing specifically what happened to total % complete, as it is reset in older journals, and updated in the last one, but it can indeed test for more. 

I added a commit to also test for ∑ remaining work in the same test.